forge-gui/res/cardsfolder/upcoming/the_seventh_doctor.txt

```diff
@@ -0,0 +1,15 @@
+Name:The Seventh Doctor
+ManaCost:3 W U
+Types:Legendary Creature Time Lord Doctor
+PT:3/6
+T:Mode$ DamageDone | ValidSource$ Card.Self | ValidTarget$ Player | CombatDamage$ True | Execute$ TrigGuess | TriggerZones$ Battlefield | TriggerDescription$ When CARDNAME attacks, choose a card in your hand. Defending player guesses whether that card's mana value is greater than the number of artifacts you control. If they guessed wrong, you may cast it without paying its mana cost. If you don't cast a spell this way, investigate. (Create a Clue token. It's an artifact with "{2}, Sacrifice this artifact: Draw a card.")
```

Comment:
wrong trigger

Reply:
when you change a trigger you need to be mindful to use the right triggeringObjects too

Reply:
`TriggeredDefender `is not correct ?

Reply:
no, use DefendingPlayer since it could also attack a Card and then you're in trouble

Reply:
it's still a triggeringObject